trio/_util.py

```diff
@@ -135,28 +135,26 @@ def _return_value_looks_like_wrong_library(value: object) -> bool:
 
             raise TypeError(
                 "Trio was expecting an async function, but instead it got "
-                "a coroutine object {async_fn!r}\n"
+                f"a coroutine object {async_fn!r}\n"
                 "\n"
                 "Probably you did something like:\n"
                 "\n"
-                "  trio.run({async_fn.__name__}(...))            # incorrect!\n"
-                "  nursery.start_soon({async_fn.__name__}(...))  # incorrect!\n"
+                f"  trio.run({async_fn.__name__}(...))            # incorrect!\n"
+                f"  nursery.start_soon({async_fn.__name__}(...))  # incorrect!\n"
```

Comment:
Mind checking the commit that these came in with (or the PR more generally) and seeing if there's other stuff like this?

Reply:
Same string concat weirdness (see other comment)